docs/tables/aws_cloudwatch_metric_statistic_data_point.md

```diff
@@ -0,0 +1,190 @@
+# Table: aws_cloudwatch_metric_statistic_data_point
+
+This table provides statistic data points for the specified metric.
+
+The maximum number of data points returned from a single call is 1,440. If you request more than 1,440 data points, CloudWatch returns an error. To reduce the number of data points, you can narrow the specified time range and make multiple requests across adjacent time ranges, or you can increase the specified period. Data points are not returned in chronological order.
+
+If you need to fetch more than 1440 data points then please use the `aws_cloudwatch_metric_data_point` table.
+
+- You **_must_** specify `metric_name`, `namespace`, `timestamp` and `dimensions` in a `where` clause in order to use this table. To fetch aggregate statistics data `dimensions` is not required.
+
+- You need to provide the timestamp value in a range like `timestamp >= '2023-03-15T00:00:00Z' and timestamp <= '2023-03-15T13:33:00Z' or like in the examples below.`
+
+- The GetMetricStatistics API used for this table cannot process multiple dimension values at a time, so you need to pass one set of dimensions to the query, like in the examples below.
```

Comment:
What happens if you hit the API (or use the AWS CLI) and pass the same dimension name twice in a single request/command?

Reply:
The API returns an empty response if we pass another/same dimension in the query-
<img width="945" alt="image" src="https://user-images.githubusercontent.com/78197905/225547530-e89efe94-271e-4d98-a152-e364bfba53c4.png">
<img width="942" alt="image" src="https://user-images.githubusercontent.com/78197905/225547590-9dbeaedc-1396-47fa-a112-a5e8cfc60221.png">

Same with the CLI as well.